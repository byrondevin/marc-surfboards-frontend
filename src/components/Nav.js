
export default function Nav() {


    return(
        <>
            {/* Bootstrap navbar */}
            <nav class="navbar navbar-expand-lg navbar-light bg-light">

                <div class="navbar" id="navbarNav">

                    <ul class="navbar-nav">

                        {/* Home link */}
                        <li class="nav-item active">

                            <a class="nav-link" href="/">Home <span class="sr-only">(current)</span></a>
                        
                        </li>

                       
                       {/* Login link */}
                        <li class="nav-item">
                        
                            <a class="nav-link" href="/login">Login</a>
                        
                        </li>


                        {/* Sign-up link */}
                        <li class="nav-item">
                        
                            <a class="nav-link" href="/signup">Sign-Up</a>
                        
                        </li>


                         {/* Enquiries link */}
                         <li class="nav-item">
                        
                            <a class="nav-link" href="/enquiries">Enquiries</a>
                        
                        </li>
                        

                        {/* Users link */}
                        <li class="nav-item">
                        
                            <a class="nav-link" href="/users">Users</a>
                        
                        </li>
                    </ul>
                </div>
            </nav>
        </>
        
    )
}