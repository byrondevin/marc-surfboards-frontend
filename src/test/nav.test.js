import Nav from '../components/Nav';
import React from 'react';
import renderer from 'react-test-renderer'; 
import "@testing-library/jest-dom";







//nav component snapshot test

test('nav matches snapshot', () => {
    const tree = renderer
  
      .create(<Nav />)
      .toJSON()
      expect(tree).toMatchSnapshot();
    
  });

