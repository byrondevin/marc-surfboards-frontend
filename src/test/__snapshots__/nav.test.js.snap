// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nav matches snapshot 1`] = `
<nav
  class="navbar navbar-expand-lg navbar-light bg-light"
>
  <div
    class="navbar"
    id="navbarNav"
  >
    <ul
      class="navbar-nav"
    >
      <li
        class="nav-item active"
      >
        <a
          class="nav-link"
          href="/"
        >
          Home 
          <span
            class="sr-only"
          >
            (current)
          </span>
        </a>
      </li>
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
          href="/login"
        >
          Login
        </a>
      </li>
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
          href="/signup"
        >
          Sign-Up
        </a>
      </li>
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
          href="/enquiries"
        >
          Enquiries
        </a>
      </li>
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
          href="/users"
        >
          Users
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
